internal/core/metadata/rest_scheduleevent.go

```diff
@@ -611,7 +614,7 @@ func restAddSchedule(w http.ResponseWriter, r *http.Request) {
 	if s.Frequency != "" {
 		if !isIntervalValid(s.Frequency) {
 			err := errors.New("Frequency format incorrect: " + s.Frequency)
-			LoggingClient.Error("Frequency format is incorrect: "+err.Error(), "")
+			LoggingClient.Error(err.Error(), "")
```

Comment:
Just a comment -- This could be a good candidate for a custom error type.

Reply:
You are right, but let's do it in another PR :)